projecttacoma/fqm-testify
# Changed date picker to date range picker

# Summary
Converted the measurement period date pickers to date range pickers which automatically validate the date range selected
## New Behavior
date pickers -> date range pickers. Should be no other noticeable changes

## Code Changes
- Removed a date picker and replaced the other date picker with a date range picker which stores both start and end of measurement period range
- minor testing updates

# Testing Guidance
- `npm run check`
- `npm run dev`
- upload a Measure Bundle and make sure that the date range picker updates as expected. Continue through the app ensuring the proper measurement period is observed
- try massing with the date range yourself and ensure it doesn't allow the range end to be before the range start.
- NOTE: Unit testing for this is brutal. LMK if you have any ideas!!
